atc/atccmd/command.go

```diff
@@ -1420,6 +1424,9 @@ func (cmd *RunCommand) configureComponentIntervals(componentFactory db.Component
 			}, {
 				Name:     atc.ComponentCollectorWorkers,
 				Interval: cmd.GC.Interval,
+			}, {
+				Name:     atc.ComponentCollectorPipelines,
+				Interval: 5 * time.Minute,
```

Comment:
Mind adding a flag for this or just using `cmd.GC.Interval`?

Reply:
I don't feel it is worthy adding a new flag as this is not a critical collector task. `cmd.GC.Interval` seems to be too frequent for this task. If collectors all use `cmd.GC.Interval`, all GC tasks might start at same time, would it cause GC storm? 

I hard coded 5 minutes here because I saw there are a couple of other components like `atc.ComponentScheduler` and `atc.ComponentBuildReaper` also hard code their intervals. Maybe that's reason we should never leave bad samples in code. :) But I feel ok for this non-critical task. Are you ok? @vito

Reply:
Changed to use `cmd.GC.Interval`.